root/static/scripts/common/components/EntityLink.js

```diff
@@ -96,9 +96,19 @@ const AreaDisambiguation = ({area}: {+area: AreaT}) => {
   return <Comment className="historical" comment={comment} />;
 };
 
+const DisabledLinkText = l(`This link has been temporarily disabled because
```

Comment:
```suggestion
const disabledLinkText = N_l(`This link has been temporarily disabled because
```

`N_l` because it's in the module scope and lower starting case since CapWords should be reserved for classes and components.